figanos/matplotlib/plot.py

```diff
@@ -2311,6 +2312,13 @@ def hatchmap(
             if "hatches" not in plot_kw[k].keys():
                 plot_kw[k]["hatches"] = dfh[n]
                 n += 1
+            elif isinstance(
+                plot_kw[k]["hatches"], str
+            ):  # make sure the hatches are in a list
+                warnings.warn(
+                    "Hatches argument must be of type 'list'. Wrapping string argument as list."
```

Comment:
But it also works without a list no? Maybe, we could specify that it's the spacing that doesn't work without a list.

Reply:
peut-être qu'il me manque de contexte, mais on devrait accepter tous les formats acceptés par matplotlib